LoadTests/SET_UP_TEST_ENTITIES.sql

```diff
@@ -1,26 +1,31 @@
 DO $$
 
     DECLARE
+        -- A high starting number to ensure that the IDs don't interfere with any existing data
         STARTING_ID INTEGER := 300000000;
         NUM_OF_USERS INTEGER := 20000;
         INDEX INTEGER := 1;
 
 -- Remove all test entities
     BEGIN
-        DELETE FROM "UserStatus" WHERE "ByUserId" = STARTING_ID + INDEX;
-        DELETE FROM "Users" WHERE "Users"."UserId" > STARTING_ID AND "Users"."UserId" <= STARTING_ID + NUM_OF_USERS;
-        DELETE FROM "Organisations" WHERE "Organisations"."OrganisationId" > STARTING_ID AND "Organisations"."OrganisationId" <= STARTING_ID + 2 * NUM_OF_USERS;
-        DELETE FROM "UserOrganisations" WHERE "UserOrganisations"."OrganisationId" > STARTING_ID AND "UserOrganisations"."OrganisationId" <= STARTING_ID + 2 * NUM_OF_USERS;
-        DELETE FROM "OrganisationScopes" WHERE "OrganisationScopes"."OrganisationScopeId" > STARTING_ID AND "OrganisationScopes"."OrganisationScopeId" <= STARTING_ID + 2* NUM_OF_USERS;
-        DELETE FROM "OrganisationNames" WHERE "OrganisationNames"."OrganisationId" > STARTING_ID AND "OrganisationNames"."OrganisationId" <= STARTING_ID + 2 * NUM_OF_USERS;
-        DELETE FROM "OrganisationSicCodes" WHERE "OrganisationSicCodes"."OrganisationId" > STARTING_ID AND "OrganisationSicCodes"."OrganisationId" <= STARTING_ID + 2 * NUM_OF_USERS;
-        DELETE FROM "OrganisationAddresses" WHERE "OrganisationAddresses"."AddressId" > STARTING_ID AND "OrganisationAddresses"."AddressId" <= STARTING_ID + 2 * NUM_OF_USERS;
-        DELETE FROM "Returns" WHERE "Returns"."OrganisationId" > STARTING_ID AND "Returns"."OrganisationId" <= STARTING_ID + NUM_OF_USERS;
+        DELETE FROM "AuditLogs" WHERE "OrganisationId" > STARTING_ID;
+        DELETE FROM "UserStatus" WHERE "ByUserId" > STARTING_ID;
+        DELETE FROM "OrganisationStatus" WHERE "OrganisationId" > STARTING_ID;
+        DELETE FROM "OrganisationStatus" WHERE "ByUserId" > STARTING_ID;
+        DELETE FROM "UserOrganisations" WHERE "UserOrganisations"."OrganisationId" > STARTING_ID;
+        DELETE FROM "OrganisationScopes" WHERE "OrganisationScopes"."OrganisationScopeId" > STARTING_ID;
+        DELETE FROM "OrganisationNames" WHERE "OrganisationNames"."OrganisationId" > STARTING_ID;
+        DELETE FROM "OrganisationSicCodes" WHERE "OrganisationSicCodes"."OrganisationId" > STARTING_ID;
+        DELETE FROM "OrganisationAddresses" WHERE "OrganisationAddresses"."AddressId" > STARTING_ID;
+        DELETE FROM "Returns" WHERE "Returns"."OrganisationId" > STARTING_ID;
 
         -- Delete users created using registration journey
         DELETE FROM "UserStatus" WHERE "ByUserId" IN (SELECT "UserId" FROM "Users" WHERE "Users"."Firstname" = 'Test' AND "Users"."Lastname" = 'Example' AND "Users"."JobTitle" = 'Tester');
         DELETE FROM "Users" WHERE "Users"."Firstname" = 'Test' AND "Users"."Lastname" = 'Example' AND "Users"."JobTitle" = 'Tester';
 
+        DELETE FROM "Users" WHERE "Users"."UserId" > STARTING_ID;

```

Comment:
I guess it doesn't - line 24 was from last year but I added line 26 since it feels like it's more likely to pick up all our test entities. But I forgot to remove the first one! Will do 😄